### PR TITLE
feat(gap): replace gap calls with our spacing tokens

### PR DESCRIPTION
**Pull request by araujobarret**
### Description

This PR bumps palette-mobile with the latest changes and adjusts the gaps to the tokens.

2 minor changes:
- removed the unnecessary gap of 1dp on the credit card screen(screenshots)
- fixed the artwork rails placeholder that was using 15px instead of our 2(20px/dp)

Before:

Simulator Screenshot - iPhone 14 Pro - 2024-10-30 at 15 33 24

After:

Simulator Screenshot - iPhone 14 Pro - 2024-10-30 at 15 33 40

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- refactor: adjust gaps to our tokens

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 913061e3e05369c30dfc852abb5a6f640f9ab941
</p>

**Review by damassi — APPROVED** [review]
Yessss 🙏

**Review by gkartalis — APPROVED** [review]

File: src/app/Components/ArtworkRail/ArtworkRail.tsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ export const ArtworkRailPlaceholder: React.FC = () => {
   const cards = !isTablet() ? 2 : 6
 
   return (
-    <Flex gap={15} flexDirection="row">
+    <Flex gap={2} flexDirection="row">
```
Comment (gkartalis) [inline src/app/Components/ArtworkRail/ArtworkRail.tsx:147]: are you sure this doesn't break anything? I don't see any screenshots

Comment (damassi) [reply src/app/Components/ArtworkRail/ArtworkRail.tsx:147]: FWIW -- stepping out of the DS is technically breaking things, not the other way around!

Comment (gkartalis) [reply src/app/Components/ArtworkRail/ArtworkRail.tsx:147]: it def makes sense, but screenshots + videos are also required for visual changes 😆

**Pull request merged**